### PR TITLE
Add empty titleSuffix to one article

### PR DESCRIPTION
**Pull request by gewarren**

### CODE REVIEW

**Review by mairaw — APPROVED** [review]
This didn't seem to have a suffix before but this works

**Comment by gewarren** [comment]
@mairaw I think I meant to change https://docs.microsoft.com/en-us/dotnet/core/porting/net-framework-tech-unavailable?toc=/dotnet/core/compatibility/toc.json&bc=/dotnet/breadcrumb/toc.json instead. But I ended up adding a global titleSuffix to this folder too. Could you take another look?

File: docs/core/compatibility/breaking-changes.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: .NET Core breaking changes
```
Comment (mairaw) [inline docs/core/compatibility/breaking-changes.md]: Should you do this instead?

```suggestion
title: Breaking changes
```

File: docs/core/compatibility/breaking-changes.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
...
+titleSuffix: ""
```
Comment (mairaw) [inline docs/core/compatibility/breaking-changes.md]:
```suggestion
```

**Review by mairaw — APPROVED** [review]
LGTM. Left a few comments.

**Pull request merged**